Automattic/simplenote-ios
# DRAFT: connect widgets to db

### Fix
This is a proof of concept connecting the core data DB to the intents extension

### Test
Not much to test at this point.  The PR only connects the shared DB to the intents extension.  So you can see that the notes are selectable but they are not yet wired into the widget it's self.

To see the intent extension in action....
1. load and run this branch
2. log into simplenote on device/simulator if not already logged in
3. minimize app and long press on home screen to bring up edit mode
4. select +, find simplenote, add one of the note preview widgets
5. once added tap on the note preview widget to see the configuration menu
6. finally select the configuration menu for note and you should see a list of the notes pulled from your core data DB

<img width="268" alt="Screen Shot 2021-06-30 at 6 10 11 PM" src="https://user-images.githubusercontent.com/22015538/123921702-675ac800-d9db-11eb-922b-e95a2aca5622.png">


### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
